### DISCUSSION TITLE
Extra Image Text File

### DISCUSSION THREAD

**Discussion by dilsg** [aaronwmorris/indi-allsky#910] [Q&A] [answered]
How does it work with the text-file in the "Extra Image Text File"? Where is it on the image?
Best regards

**Comment by aaronwmorris** [comment, marked as answer]
Technically, the text is started at the end (the location) after the last text in the label.  However, you should be able to include the same location and color overrides as the regular label data.

https://github.com/aaronwmorris/indi-allsky/wiki/Image-Labels

**Comment by dilsg** [comment]
Thank you. It is working now. I wrote "# x: 100,100" and then it was within the image.
